config/datastores/formatters.yaml

```diff
@@ -23,6 +23,7 @@ ObjectMaskCatalog: lsst.pipe.tasks.objectMasks.RegionFileFormatter
 DataFrame: lsst.daf.butler.formatters.parquetFormatter.ParquetFormatter
 Defects: lsst.daf.butler.formatters.fitsCatalogFormatter.FitsCatalogFormatter
 QECurve: lsst.daf.butler.formatters.fitsCatalogFormatter.FitsCatalogFormatter
+CrosstalkCalib: lsst.daf.butler.formatters.fitsCatalogFormatter.FitsCatalogFormatter
```

Comment:
Once you rebase you'll have to change this to `lsst.obs.base.fitsGenericFormatter.FitsGenericFormatter`

Reply:
Definitely going to defer to the gen3 expert on this one 😄